.github/workflows/build.yml

```diff
@@ -49,18 +49,18 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
         with:
-          key: cargo-${{ inputs.python-architecture }}-${{ inputs.os }}-${{ inputs.msrv }}
+          key: cargo-${{ inputs.python-architecture }}
         continue-on-error: true
 
       - if: inputs.os == 'ubuntu-latest'
         name: Prepare LD_LIBRARY_PATH (Ubuntu only)
         run: echo LD_LIBRARY_PATH=${pythonLocation}/lib >> $GITHUB_ENV
 
-      - if: inputs.msrv == 'MSRV'
+      - if: inputs.rust == '1.56.0'
```

Comment:
I removed the "msrv" input as a bit of simplification. It meant this one line gets coupled to our msrv, but I think that's trivial to replace when future msrv bumps come along.